### ISSUE TITLE
Error while using template module in Ansible Playbooks - creation and execution

### ISSUE THREAD

**Issue by THRINADH43** [spurin/diveintoansible-lab#90]
Following the content in the course,
Able to solve till now but unable to solve from creating a task in the template module
Please look at the attachment for more reference.
I tried checking indentation but didn't work.
Please help me in solving this.
Thanks in advance
![ansible playbook error 1](https://user-images.githubusercontent.com/74821042/154487293-5404f8cd-210e-49b3-b370-aebec5fe21e7.jpg)
![ansible playbook error](https://user-images.githubusercontent.com/74821042/154487323-63b62781-29b8-4de2-8db0-afeb26a141ff.jpg)
.

**Comment by spurin** [comment]
Hi @THRINADH43 

Nice to see you're progressing so well with the course!  You're taking the challenge so, what is needed in this is to popular the group_vars

If you have a look in one of the previous revision's, you should just be able to recursively copy that directory and you'll be good.

Once done, have a look inside at the files and you'll see that the variables needed will be inside.

Hopefully this helps, don't hesitate to follow up if there's more problems.

Best Regards


James Spurin

**Comment by THRINADH43** [comment]
Hi @spurin,
The problem is resolved but I am constantly facing an issue.
In all the hosts config files centos1 is configured to port 222.
but To make this happen as discussed in the previous videos. If I change the uncomment to port 2222 line the docker-compose.yaml file then I am unable to run the centos1 and portal containter stating an error that " It is pre- occupied".
Any suggestion on to change this? 

and after running the current lesson.
I got the new error.
![ansible playbook error 2](https://user-images.githubusercontent.com/74821042/154518223-6268a93a-9bf8-4a96-8846-3a74cd70e0d7.jpg)
Is this because I changed the port to 22 from 2222 in hosts_var from centos1 or due to something else?
Thanks in advance.
with regards,
Thrinadh
